vocabularies.md

```diff
@@ -0,0 +1,283 @@
+# Controlled Vocabularies Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+All the endpoints here described are restricted to authenticated users.
+
+## Main Endpoint
+**/api/submission/vocabularies**   
+
+Provide access to the configured controlled vocabularies. It returns the list of existent controlled vocabularies.
+
+Example:
+```json
+{
+  "_embedded": {
+    "vocabularies": [
+      {
+        "id": "srsc",
+        "name": "srsc",
+        "scrollable": false,
+        "hierarchical": true,
+        "preloadLevel": 1,
+        "type": "vocabulary",
+        "_links": {
+          "entries": {
+            "href": "https://dspace7.4science.cloud/server/api/submission/vocabularies/srsc/entries"
+          },
+          "self": {
+            "href": "https://dspace7.4science.cloud/server/api/submission/vocabularies/srsc"
+          }
+        }
+      },
+      {
+        "id": "common_types",
+        "name": "common_types",
+        "scrollable": true,
+        "hierarchical": false,
+        "type": "vocabulary",
+        "_links": {
+          "entries": {
+            "href": "https://dspace7.4science.cloud/server/api/submission/vocabularies/srsc/entries"
+          },
+          "self": {
+            "href": "https://dspace7.4science.cloud/server/api/submission/vocabularies/srsc"
+          }
+        }
+      },
+      {
+        "id": "common_iso_languages",
+        "name": "common_iso_languages",
+        "scrollable": true,
+        "hierarchical": false,
+        "type": "vocabulary",
+        "_links": {
+          "entries": {
+            "href": "https://dspace7.4science.cloud/server/api/submission/vocabularies/common_iso_languages/entries"
+          },
+          "self": {
+            "href": "https://dspace7.4science.cloud/server/api/submission/vocabularies/common_iso_languages"
+          }
+        }
+      }
+    ]
+  },
+  "_links": {
+    "self": {
+      "href": "https://dspace7.4science.cloud/server/api/submission/vocabularies"
+    }
+  },
+  "page": {
+    "size": 20,
+    "totalElements": 3,
+    "totalPages": 1,
+    "number": 0
+  }
+}
+```
+
+## Single Vocabulary
+**/api/submission/vocabularies/<:vocabulary-name>**
+
+Provide detailed information about a specific controlled vocabulary. The JSON response document is as follow
+```json
+{
+  "id": "srsc",
+  "name": "srsc",
+  "scrollable": false,
+  "hierarchical": true,
+  "preloadLevel": 1,
+  "type": "vocabulary"
+}
+```
+
+Attributes
+* id: the id of the vocabulary, it is the same than the name
+* name: see id
+* scrollable: if true mean that it is possible to scroll all the entries in the authority without providing a filter parameter, see (vocabulary entries)[vocabularies.md#vocabulary-entries]
```

Comment:
Can you fix this link, it's currently not a link, and I assume it should link to controlled-vocabulary-entries

Reply:
thanks, fixed

Reply:
you need to use square brackets first. You can always check this on https://github.com/4Science/Rest7Contract/blob/controlled_vocabularies/vocabularies.md#single-vocabulary and verify whether clicking the link works

Reply:
oh I haven't noted that in your first comment sorry. It works now